app/services/oidc_jwt_token_validate.rb

```diff
@@ -0,0 +1,38 @@
+class OidcJwtTokenValidate
+  attr_reader :decoded_token
+  delegate :logger, to: :Rails
+
+  def initialize(jwt_token)
+    @jwt_token = jwt_token
+  end
+
+  def valid_jwt?
+    return false if (Setting['oidc_algorithm'] == 'none' || Setting['oidc_algorithm'].nil?)
```

Comment:
Don't we typically use symbols in `Setting`?
```suggestion
    return false if (Setting[:oidc_algorithm] == 'none' || Setting[:oidc_algorithm].nil?)
```

Reply:
Looking at other files in the `app/service/sso`(where most of my code is), we have used strings in `Setting` (e.g in apache.rb, oauth.rb or jwt.rb) but as I can see in the console, I get the required value in strings and well as in the symbol form.